.travis.yml

```diff
@@ -113,11 +113,10 @@ script:
     - export PYTHONPATH=/tmp/$TRAVIS_JOB_ID/lib/python$TRAVIS_PYTHON_VERSION/site-packages
     # install easybuild-framework to unique temporary location using prepared sdist tarball
     - mkdir -p $PYTHONPATH; easy_install --prefix /tmp/$TRAVIS_JOB_ID $TRAVIS_BUILD_DIR/dist/easybuild-framework*tar.gz
-    # actively break "import setuptools" and "import pkg_resources",
+    # make sure there are no "import setuptools" or "import pkg_resources" statements,
     # since EasyBuild should not have a runtime requirement on setuptools
-    - echo 'raise ImportError("setuptools should no longer be relied on")' > /tmp/$TRAVIS_JOB_ID/setuptools.py
-    - echo 'raise ImportError("setuptools is not available")' > /tmp/$TRAVIS_JOB_ID/pkg_resources.py
-    - export PYTHONPATH=/tmp/$TRAVIS_JOB_ID:$PYTHONPATH
+    - SETUPTOOLS_IMPORTS=$(egrep -RI '^(from|import) pkg_resources|^(from|import) setuptools' easybuild/ || true)
```

Comment:
To make this 100% correct you need
'^ *(from|import) *pkg_resources|^ *(from|import) *setuptools'
and in fact it should be '^[ \t]*...'
where \t is an actual tab.

Reply:
No, I only want to catch top-level imports, so the `^(` is deliberate.

Otherwise we would also hit the `from pkg_resources import ...` in `gc3pie.py`.

You're right about the wildcard before `pkg_resources` though I guess, in theory...

Reply:
Ah yes, i keep forgetting that python is indentation language.